lib/ts/recipe/session/utils.ts

```diff
@@ -114,6 +114,8 @@ export function validateAndNormaliseUserInput(
         sessionRefreshFeature.disableDefaultImplementation = config.sessionRefreshFeature.disableDefaultImplementation;
     }
 
+    let enableAntiCsrf = config === undefined || config.enableAntiCsrf === undefined ? false : config.enableAntiCsrf;
```

Comment:
This should not be set to `false` by default. It should be set based on the value of sameSite